ajax/libs/material-components-web/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "material-components-web",
+  "version": "0.11.1",
+  "filename": "material-components-web.min.js",
+  "description": "Modular and customizable Material Design UI components for the web",
+  "homepage": "https://material.io/components/",
+  "keywords": [
+    "material components",
+    "material design"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Travis Kaufman <@traviskaufman>",
```

Comment:
👋 Hey! FWIW, We worked with Addy to port this over from the Original [Material Design Lite](https://github.com/google/material-design-lite) codebase, which would explain his activity on the repo. However, he doesn't work on material-components-web, so you can feel free to leave me as the author. I can also try and get you a more "official" author name, e.g. "The Material Design Team" or whatnot.

Sorry for the drive-by comment and thanks for adding MDC-Web to cdnjs!